roles/lmod/templates/lmod-redhat.j2

```diff
@@ -31,7 +27,7 @@ if ( ! $?MODULEPATH_ROOT ) then
     setenv MODULEPATH_ROOT      "{{ sm_module_root }}"
```

Comment:
Is `sm_module_root` defined somewhere? It should have a default set in the role

Reply:
working on it.